ui/k-info-panel/show_telemetry_int_data.kytos

```diff
@@ -0,0 +1,522 @@
+                                    </th>
+                                    <th>
+                                        <input v-model="EVCTextFilter[6][1]"></input>
+                                    </th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr v-for="(item, index) in filtered_EVCData">
+                                    <td><k-checkbox v-model:model="selectedEVCs" :value="item.id"></k-checkbox></td>
+                                    <td>{{item.id}}</td>
+                                    <td>{{item.name}}</td>
+                                    <td>{{item.active}}</td>
+                                    <td>{{item.enabled}}</td>
+                                    <td :class="statusColor(item)">{{item.status}}</td>
+                                    <td>{{item.status_reason}}</td>
+                                    <td>{{item.status_updated_at}}</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </k-accordion-item>
+            </k-accordion-item>
+        </k-accordion>
+    </div>
+</template>
+<script>
+    module.exports = {
+        props: ["content"],
+        data: function() {
+            return {
+                EVCTextFilter: [
+                    ["id", ""],
+                    ["name", ""],
+                    ["active", ""],
+                    ["enabled", ""],
+                    ["status", ""],
+                    ["status_reason", ""],
+                    ["status_updated_at", ""]
+                ],
+                selectedEVCs: [],
+                forceOption: [],
+                sortedColumnName: ["", 0],
+                sortIdentifier: ["", "", "", "", "", "", ""]
+            }
+        },
+        methods: {
+            statusColor(data) {
+            /**
+             * @param {Object} data - The normal table row object containing:
+             * @param {string} data.status - The status of the device the Object references.
+             * @returns {string} Returns a string that acts as the class for current HTML element.
+             * @description Sets color for current status table cell.
+             */
+                switch(true) {
+                    case (data.status == "UP"):
+                        return "green";
+                    case (data.status == "DOWN"):
+                        return "red";
+                }
+            },
+            //Changes the current column to sort the table by
+            changeSortedColumn(name) {
+                if (this.sortedColumnName[0] == name && this.sortedColumnName[1] == 0) {
+                    this.sortedColumnName[1] = 1
+                } else {
+                    this.sortedColumnName = [name, 0]
+                }
+                switch(name) {
```

Comment:
Heriberto, we can generalize the cases of this `switch` here with two methods:

1 - A new method to take `name` and then set `this.sortIdentifier` accordingly. You can use a JS object or a hashmap mapping the cases to witch column id they need to use like: `{ID: 0, NAME: 1, ACTIVE: 2, ...}` and so on. 
2 - A new helper method to generate an equivalent sortIdentifier list filled with empty values and set the string `"▼"` or `"▲"` on the given index, and then you can reuse this with the method mentioned on the prior item here, so then you can genererate all of the expected combinations of the possible index dynamically. 

See what I mean? The current code works but it can be more concise, and even to refactor in the future if we add new columns it'll be easier too.